docs/src/main/paradox/scala/http/common/marshalling.md

```diff
@@ -15,24 +15,24 @@ negotiate the result content types based on the @unidoc[Accept] and the `AcceptC
 
 ## Basic Design
 
-Marshalling of instances of type `A` into instances of type `B` is performed by a @scala[@unidoc[Marshaller[A, B]`]@java[@unidoc[Marshaller[A, B]]].
+Marshalling of instances of type `A` into instances of type `B` is performed by a @scala[@unidoc[Marshaller[A, B]]]@java[@unidoc[Marshaller[A, B]]].
 
-Contrary to what you might initially expect, @scala[`Marshaller[A, B]]]@java[@unidoc[Marshaller[A, B]]] is not a plain function `A => B` but rather
+Contrary to what you might initially expect, @scala[@unidoc[Marshaller[A, B]]]@java[@unidoc[Marshaller[A, B]]] is not a plain function `A => B` but rather
 essentially a function @scala[`A => Future[List[Marshalling[B]]]`]@java[`A => CompletionStage<List<Marshalling<B>>>`].
 Let's dissect this rather complicated looking signature piece by piece to understand why marshallers are designed this
 way.
-Given an instance of type `A` a @scala[@unidoc[Marshaller[A, B]`]@java[@unidoc[Marshaller[A, B]]] produces:
+Given an instance of type `A` a @scala[@unidoc[Marshaller[A, B]]]@java[@unidoc[Marshaller[A, B]]] produces:
 
 1. A @scala[`Future`]@java[`CompletionStage`]: This is probably quite clear. Marshallers are not required to synchronously produce a result, so instead
 they return a future, which allows for asynchronicity in the marshalling process.
 
 2. of `List`: Rather than only a single target representation for `A` marshallers can offer several ones. Which
 one will be rendered onto the wire in the end is decided by content negotiation.
-For example, the @scala[`Marshaller[OrderConfirmation, MessageEntity]`]@java[@unidoc[Marshaller[OrderConfirmation, MessageEntity]]] might offer a JSON as well as an XML representation.
+For example, the @scala[@unidoc[Marshaller[OrderConfirmation, MessageEntity]]]@java[@unidoc[Marshaller[OrderConfirmation, MessageEntity]]] might offer a JSON as well as an XML representation.
 The client can decide through the addition of an @unidoc[Accept] request header which one is preferred. If the client doesn't
 express a preference the first representation is picked.
 
-3. of @scala[`Marshalling[B]]]@java[`Marshalling<B>`]: Rather than returning an instance of `B` directly marshallers first produce a
+3. of @scala[`Marshalling[B]`]@java[`Marshalling<B>`]: Rather than returning an instance of `B` directly marshallers first produce a
```

Comment:
Could that be unidoc?

Reply:
It seems we have only `akka.http.scaladsl.marshalling.Marshalling` and use that for Java as well (despite the package name), is that correct? The directive is conservative and doesn't support generating a javadoc link in that case, though I guess we could add a 'whitelist'?

Reply:
Guess, it's fine for now, let's add a ticket to improve it later.